### DISCUSSION TITLE
Allow edits and access to secrets by maintainersを強制する？

### DISCUSSION THREAD

**Discussion by KisaragiEffective** [misskey-dev/misskey#13973] [Ideas]
Pull RequestにおいてAllow edits and access to secrets by maintainersは強制されるべきかどうかについて議論を行いたいと思います。

文脈については https://github.com/misskey-dev/misskey/pull/13390#issuecomment-1974748085 や https://github.com/misskey-dev/misskey/pull/13967#issuecomment-2164194196 などをご覧ください。

個人的には強制とまで行くと微妙ですがほのめかす程度ならしても良いのではないかと思います。その理由としてcontributor側のブランチにプッシュされても普通は知るすべがないこと、suggestionで明記するほうがデザインノート代わりとなることなどがあげられます。

**Comment by anatawa12** [comment]
ソースがorganizationだとダメって話があった気がするので強制は難しい気がする (#13967 がreopenになったのもこれが理由だと思われる)

**Comment by kakkokari-gtyih** [comment]
Pull Request Templateに書くくらいはやっても良さそう

**Reply by KisaragiEffective** [reply]
テンプレートに書いたところで結局チェックしない人はチェックしないです。私もたまにやってしまいます (後で直すことがほとんどです)。
APIから有効になってるか見て無効化されていたら[^1]お願いする形の方がモアベターと思っています。
[^1]: デフォルトでは有効のはず

**Reply by kakkokari-gtyih** [reply]
できるならそのほうが尚良しではありますね

**Comment by kakkokari-gtyih** [comment]
https://github.com/misskey-dev/misskey/pull/15208
